test/timearray.jl

```diff
@@ -497,21 +497,20 @@ end
 │ 2000-01-10 │ 97.75  │
 │ 2000-01-11 │ 92.75  │
 │ 2000-01-12 │ 87.19  │
-│ 2000-01-13 │ 96.75  │
-   ⋮
-│ 2001-12-19 │ 21.62  │
+│ ⋮          │ ⋮      │
 │ 2001-12-20 │ 20.67  │
 │ 2001-12-21 │ 21.0   │
 │ 2001-12-24 │ 21.36  │
 │ 2001-12-26 │ 21.49  │
 │ 2001-12-27 │ 22.07  │
 │ 2001-12-28 │ 22.43  │
-│ 2001-12-31 │ 21.9   │"""
+│ 2001-12-31 │ 21.9   │
+"""
         @test str == out
     end
 
     # my edits above seem to work -- now need to do for the rest, JJS 2/22/19
-    let str = sprint(show, ohlc)
+    let str = sprint(show, ohlc; context = :limit=>true)
```

Comment:
Then, this case is just the `Base.summary`
```suggestion
    let str = sprint(summary, ohlc)
```

Reply:
You're right, I think it works better with `summary`. DataFrames do this too. I'll review the test suite with some changes on this front and let's see how it goes.